.github/workflows/analysis.yml

```diff
@@ -49,4 +49,40 @@ jobs:
           # summary
           echo "Full report is included in build Artifacts"
           echo
-          ./scripts/count-bugs.py report/*/index.html
+          ./scripts/count-bugs.py 92 report/*/index.html
+
+  dynamic_matrix:
+    name: ${{ matrix.compiler }} dynamic sanitizers
```

Comment:
Yes, if we're ok with not exercising a specific compiler.   I have a feeling we will atleast want to exercise three (if available): oldest, default, and latest.  So something as low as 4.8 or 5.4 and up to 9 or soon 10.

Reply:
I would be ok with defaults only +1 job with the newest compiler on the newest available distro.

Default on Ubuntu 16.04 is GCC 5.4, and by the time it will be EOL, a newer compiler should be available for SteamOS virtual environments (and SteamOS is literally the oldest configuration, that is being realistically exercised by users).

Reply:
Will try for this!

Reply:
Fixed in d28cc6bb; will see if that covers us.